### PR TITLE
Fix #164 #176

### LINKED ISSUE #164: Only show "Search More" when there actually is "More"
Currently the "Search More" button also shows up, even when there are not more results. Clicking on the button results in no action.

Please only show "Search More" when more results are available.

### PR DESCRIPTION
**Pull request by uttu357**
- Show loading icon after user clicks on "Show More"
- Show "Search More" only when more results are available
It has been implemented as "Show More" is not shown if the previous search request did not return the maximum records i.e set to 30.

screenshot from 2017-01-08 05 01 29
Screenshot during loading.

Link for live testing : https://uttu357.github.io/loklak_search/

Closes #164 #176

### CODE REVIEW

File: src/app/feed/feed.component.html [hunk 1]
```diff
@@ -13,8 +13,8 @@
 				<feed-card [feedItem]="item"></feed-card>
 			</div>
 
-			<feed-pagination
-				(paginate)="loadMoreResults(event$)"></feed-pagination>
+			<feed-pagination (paginate)="loadMoreResults(event$)"
```
Comment (hemantjadon) [inline src/app/feed/feed.component.html]: Pass each attribute in their respective lines just for code consistency

Comment (uttu357) [inline src/app/feed/feed.component.html]: Ohk..

**Comment by uttu357** [comment]
Yeah i thought as we were requesting for 30 records, if less than 30 records are received then it means that no more tweets are there currently. Your suggestions?

**Comment by hemantjadon** [comment]
Well truly speaking I don't know the perfect approach as there can be a situation when initially we have 10 tweets then after sometime there are new results available. I don't know maybe we can handle that situation later, I think for now we can go with this approach.

**Comment by uttu357** [comment]
@hemantjadon Made the fix. Yeah it can happen, but i could not see how we can detect that.

**Comment by hemantjadon** [comment]
Can you squash the commits? 
Yeah lets go with this for now.

**Comment by uttu357** [comment]
@hemantjadon Done.. :)

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**